### PR TITLE
add the clock and user message

### PR DESCRIPTION
**Pull request by SMcIlroy11**
I created the clock and the user message. The clock is currently in the H:mm:ss format so that I could see it was updating every second, but this can change to reflect H:mm. I hard coded my name into the user greeting, but this will need to be changed going forward as the user should be able to enter in his/her name in the settings.

I initially created modules for the different atomic features of the clock (welcome message, clock face and settings), but have chosen to do all this work together in the clock module as doing it piecemeal could cause problems later on. I will be deleting the uneeded modules.

### CODE REVIEW

File: src/_modules/clock/clock.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    let amPM = null;
+    let currentTime1 = (moment().format('H'));
+    let amTime = 12
+    if (currentTime1 >amTime) {
+      amPM = "PM";  
+    } 
+    else {
+      amPM = "AM";
+    }
+        return amPM;
```
Comment (brandon-hedrick) [inline src/_modules/clock/clock.js]: <= remove this indent

File: src/_modules/clock/clock.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    if (currentTime >= afternoonTime && currentTime < eveningTime) {
+      greeting = "Afternoon";  
+    } 
+    else if(currentTime >=eveningTime) {
+      greeting = "Evening";
+    }
+    else {
+      greeting = "Morning";
+    }
+        return greeting;
```
Comment (brandon-hedrick) [inline src/_modules/clock/clock.js]: <= remove indent

File: src/_modules/clock/clock.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    console.log(userGreeting1());
+ 
+  //updating greeting - Steph is in there as a placeholder for now
+  var greetingP = document.getElementById('pGreeting');
+    greetingP.innerText='Good ' + userGreeting1() + ' , Steph!'
```
Comment (brandon-hedrick) [inline src/_modules/clock/clock.js]: <= remove indent

File: src/index.jade [hunk 4]
```diff
@@ -20,4 +24,6 @@ block content
   .container
       +todo-list('cool')
       p do not know how to use jade
-      ul(id='todo-list')
\ No newline at end of file
+      ul(id='todo-list')
+      +clock();
```
Comment (mikepawlowski) [inline src/index.jade]: Blame Brandon for this one ;)

I'm sure you'll figure this once you get back to work after merging Matt's code, but this should be above the to-do

File: src/_modules/clock/clock.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  return amPM;
+}
+
+  //function to update the clock - adds in AM and PM from the above function
+function update() {
+  var clock = document.getElementById('clockText');
+    //set clock format with seconds so that we can see that it is updating every second
+  clock.innerText = (moment().format('H:mm:ss')) + (getAmPm());
+}
+setInterval(update, 1000);
```
Comment (mikepawlowski) [inline src/_modules/clock/clock.js]: Your clock isn't going to show up at page load

**Pull request closed without merging**